tools/project_manager/remote_file_manager.php

```diff
@@ -268,172 +252,33 @@ function do_showupload()
-        'js_data' => get_resumablejs_loader($curr_relpath),
-    );
-    output_header($page_title, NO_STATSBAR, $extra_args);
+    output_header($page_title, NO_STATSBAR, get_upload_args());
+
     echo "<h1>$page_title</h1>\n";
 
     $form_content = "";
-    if (get_access_mode($pguser) == 'common') {
+    if (get_access_mode($pguser) == 'common')
+    {
         $form_content .= get_message('info', $autoprefix_message);
     }
-    $form_content .= "<p style='margin-top: 0em;'>$standard_blurb</p>\n";
-
-    $form_content .= "<div id='old_uploader'>";
-    $form_content .= $submit_blurb;
-    $form_content .= _("File to upload") . ":&nbsp;";
-    $form_content .= "<input type='file' accept='.zip' name='the_file' size='50' maxsize='50'>";
-    $form_content .= "</div>";
-
-    $form_content .= "<div id='resumable_uploader' style='display: none;'>";
-    $form_content .= "<input type='hidden' name='resumable_filename' value=''>";
-    $form_content .= "<p>" . _("Your browser supports uploading large files (up to 1GB) via javascript. If the upload fails, or you navigate away from this page before it finishes, uploading the file again will pick up where it left off.") . "</p>";
-    $form_content .= "<p>" . _("File to upload") . ": <span id='resumable_selected_file'></span> &nbsp; ";
-    $form_content .= "<span id='resumable_browse' class='button'>" . _("Choose File") . "</span>";
-    $form_content .= "</p>";
-    $form_content .= "<p><span id='resumable_submit' class='button'>" . _("Upload") . "</span></p>";
-    $form_content .= "<p>" . _("Progress") . ": <span id='resumable_progress'></span></p>";
-    $form_content .= "</div>";
-
-    show_form(
-        'upload',
-        $curr_relpath,
-        $form_content,
-        _("Upload")
-    );
+    $form_content .= "
+        <input type='hidden' name='cdrp' value='" . attr_safe($curr_relpath) . "'>\n
+        <input type='hidden' name='action' value='upload'>\n";
 
+    show_upload_form($form_content, _("Upload"));
     show_return_link();
 }
 
 function do_upload()
 {
-    return handle_file_upload(@$_FILES['the_file']);
-}
-
-function do_resumable_final()
-{
-    // staging directory for resumable uploads
-    $root_staging_dir = "/tmp/resumable_uploads";
-    $filename = array_get($_POST, "resumable_filename", "");
-    $hashed_filename = md5($filename);
-    $uploaded_path = "$root_staging_dir/$hashed_filename/$hashed_filename";
-
-    $file_info = array(
-        "error" => UPLOAD_ERR_OK,
-        "tmp_name" => $uploaded_path,
-        "size" => filesize($uploaded_path),
-        "name" => $filename,
-        "source" => "resumable",
-    );
-    return handle_file_upload($file_info);
-}
-
-function handle_file_upload($file_info)
-{
-    global $curr_abspath, $hce_curr_displaypath, $antivirus_executable;
+    global $curr_abspath, $hce_curr_displaypath;
     global $pguser, $despecialed_username;
     global $commons_dir;
 
+    slim_header();
```

Comment:
This page should have some sort of title.

Reply:
I've given it the same title as the show_upload page now.